modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -90,6 +142,30 @@ definition {
 		MenuItem.clickNoError(menuItem = "Delete");
 	}
 
+	macro editCategoryOption {
+		Type(
+			locator1 = "TestrayManage#SELECT_CATEGORIES_IN_OPTIONS",
+			value1 = "${categorie}");
```

Comment:
Hi @joelrocha68 would you please correct this word? "categorie" must be changed to "category"